filesystems/blob/src/test/scala/cromwell/filesystems/blob/AzureFileSystemSpec.scala

```diff
@@ -0,0 +1,25 @@
+package com.azure.storage.blob.nio;
```

Comment:
The intent for this is to live with the NIO library, but was struggling to get Scala tests working as a part of that library